docs/installation/020_INSTALL_MANUAL.md

```diff
@@ -208,7 +157,7 @@ manager, please follow these steps:
     time you try to generate an image. Fortunately, after the warm-up period
     rendering will be fast.
 
-10. Subsequently, to relaunch the script, be sure to enter `InvokeAI` directory,
+9.  Subsequently, to relaunch the script, be sure to enter `InvokeAI` directory,
```

Comment:
User does not need to be in the InvokeAI directory, as the command will be in the path.

Reply:
Did I even touch this? xD

I only wanted to move out the Part which is pointing to the requirement files, which should maybe also get delted in #2448 - since we stopped using them when switching to pyproject.toml, just to prevent that in a week the next PR comes in to re-create the requirements installation steps 🙈

So I would still like to wait with the docs till we know we merge #2448 as it is and hope that @ebr will also provide docs for the changes which are introduced with it (which would basically mean that changes here will become mostly obsolete)